dev-itpro/administration/administration-center-api.md

```diff
@@ -27,7 +27,99 @@ For more information about administrative capabilities, see [The Business Centra
 
 The [!INCLUDE[prodadmincenter](../developer/includes/prodadmincenter.md)] API is located at the following URL: https://api.businesscentral.dynamics.com.
 
-## <a name="azuread"></a>Setting up Azure Active Directory (Azure AD) based authentication
+## Authenticate using service-to-service AAD Apps ([Client Credentials Flow](/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow))
```

Comment:
something strange is with this link, maybe because it points to different docs than BC, but gives me 404